index.html

```diff
@@ -0,0 +1,44 @@
+<link rel="stylesheet" href="stylesTributePage.css">
+
+<style>
+   body {
+      background-color: rgb(90, 0, 80);
+      color: white;
+      font-family: calibri;
+   }
+</style>
+
+<html lang="en">
+
+<head>
+   <title>Tribute Page</title>
+</head>
+
+<body>
+
+   <div id="main">
+      <h1 id="title">Sevagoth - The slayer</h1>
+
+      <div id="img-div">
+         <a id="tribute-link" target="_blank" href="https://warframe.fandom.com/wiki/Sevagoth">
+            <img id="image"
+               src="https://www.ggrecon.com/media/x4tlnglf/warframe-sevagoth-2.jpg?mode=crop&width=682&quality=80&format=webp"
+               alt="Super duper cool Sevagoth image" width="100%"></a>
+         <figcaption id="img-caption"> The peak of power, strength and efficiency . Razer sharp claws with the
+            capabilaty of slicing foe with zero effort!</figcaption>
+      </div>
+
+      <p id="tribute-info">
+         <b>Sevagoth's abilities</b>
+      <ul>
+         <li>Reap</li>
+         <li>Sow</li>
+         <li>Gloom</li>
+         <li>Exalted Shadow</li>
+      </ul>
+      </p>
+
+   </div>
+</body>
+
+</html>
```

Comment:
In the html file, the are too many empty lines between code. This makes the code look unprofessional.

Reply:
- [ ] Dear @Thinus01, Good job providing the feedback with explanation on why. As I have mention is the other comment, you can Improve this further by making your reviews polite, by giving appreciation for what have been don good so far and adding `Please`, `kindly`, `to make it better`  or other terms that sound suggestion than order. This allows you to sound more friendly in your comment. you could use a format like `good job doing this`, then `pleas or kindly apply this`  then `the reason for your suggestion`